system/datatypes/peer_reviewed_article.php

```diff
@@ -954,262 +663,444 @@ public function discussionClosedAction($discussionId) {
+				<div class="input-wrapper field_type_text field_name_[[authorFieldName]]">
+					<strong><label for="[[authorFieldName]]">[[author]]</label></strong><br><input type="text" id="[[authorFieldName]]" name="[[authorFieldName]]" />
 				</div>
 			</div>
 			<div class="section" style="padding:5px; margin-bottom:5px; position:relative;">
-				<div class="input-wrapper field_type_editor field_name_$textFieldName">
-					<strong><label for="$textFieldName"> $text </label></strong>$infoArtText<br><editor name="$textFieldName"></editor>
+				<div class="input-wrapper field_type_editor field_name_[[textFieldName]]">
+					<strong><label for="[[textFieldName]]">[[text]]</label></strong>[[infoArtText]]<br><editor name="[[textFieldName]]"></editor>
 				</div>
 			</div>
 			<div class="section" style="padding:5px; margin-bottom:5px; position:relative;">
-				<div class="input-wrapper field_type_editor field_name_$excerptFieldName">
-					<strong><label for="$excerptFieldName"> $excerpt </label></strong> $infoArtExcerpt <br><editor name="$excerptFieldName"></editor>
+				<div class="input-wrapper field_type_editor field_name_[[excerptFieldName]]">
+					<strong><label for="[[excerptFieldName]]">[[excerpt]]</label></strong>[[infoArtExcerpt]]<br><editor name="[[excerptFieldName]]"></editor>
 				</div>
 			</div>
 			<div class="section" style="padding:5px; margin-bottom:5px; position:relative;">
-				<div class="input-wrapper field_type_editor field_name_$methodFieldName">
-					<strong><label for="$methodFieldName"> $method </label></strong>$infoArtMethod <br><editor name="$methodFieldName"></editor>
+				<div class="input-wrapper field_type_editor field_name_[[methodFieldName]]">
+					<strong><label for="[[methodFieldName]]">[[method]]</label></strong>[[infoArtMethod]]<br><editor name="[[methodFieldName]]"></editor>
 				</div>
 			</div>
 			<div class="section" style="padding:5px; margin-bottom:5px; position:relative;">
-				<div class="input-wrapper field_type_editor field_name_$sourcesFieldName">
-					<strong><label for="$sourcesFieldName"> $sources </label></strong><br><editor name="$sourcesFieldName"></editor>
+				<div class="input-wrapper field_type_editor field_name_[[sourcesFieldName]]">
+					<strong><label for="[[sourcesFieldName]]">[[sources]]</label></strong><br><editor name="[[sourcesFieldName]]"></editor>
 				</div>
 			</div>
 EOF;
-		// buttons
-		$commands = $this->findBySelector('#pageCommands');
-		$commands->append($this->makeActionButton(__('art-cancel')));
-		$commands->append($this->makeActionButton(__('art-save'), self::PATH_EDIT, self::FORM_CMD_EDIT));
-		$commandsAtEnd = $this->findBySelector('#pageEndCommands');
-		$commandsAtEnd->append($this->makeActionButton(__('art-cancel')));
-		$commandsAtEnd->append($this->makeActionButton(__('art-save'), self::PATH_EDIT, self::FORM_CMD_EDIT));
 
-		return $this->createForm($html, $data);
+		$vars = [
+			'title' => __('art-title'),
+			'titleFieldName' => self::FIELD_NAME_TITLE,
```

Comment:
Maybe this should be:

   'field-name-title' => self::FIELD_NAME_TITLE,

Then both sides are a bit more consistent with each other? If so, we should change that in text as well (I'll do that, then).

Maybe also use the same syntax (dash-separated parts rather than camelCase) for the other variables?

Reply:
done